run-ccsp2-build

```diff
@@ -19,12 +19,14 @@ export METRICS_UTILITY_REPORT_RHN_LOGIN="test_login"
 export METRICS_UTILITY_REPORT_SKU="MCT3752MO"
 export METRICS_UTILITY_REPORT_SKU_DESCRIPTION="EX: Red Hat Ansible Automation Platform, Full Support (1 Managed Node, Dedicated, Monthly)"
 
-/usr/bin/time --format='\n\nmemory %MK\ncpu %P\ntime %es\n\n' \
+TIME=`which gtime && echo 'gtime' || echo 'time'`
+
+$TIME --format='\n\nmemory %MK\ncpu %P\ntime %es\n\n' \
 uv run ./manage.py build_report --month=2024-04 --force "$@"
 
 set -x
 ls -l metrics_utility/test/test_data/reports/2024/04/
 
 if which xlsx2csv; then
   xlsx2csv -a ./metrics_utility/test/test_data//reports/2024/04/CCSPv2-2024-04.xlsx
-fi
+fi
```

Comment:
@ShaiahWren you may need to fix your editor config - it keeps removing the last newline in each file you edit.

If you're on vscode, it should be called something like "insert final newline"